code/controllers/master_controller.dm

```diff
@@ -173,6 +177,87 @@ datum/controller/game_controller/proc/cachedamageicons()
 					damage_icon_parts["[damage_state]/[O.icon_name]/[species_blood]"] = DI
 	del(H)
```

Comment:
You should rather feel terror at how long it must have taken me to make all those dmi.